enzyme/test/Integration/ForwardError/binops.c

```diff
@@ -11,6 +11,14 @@ double fabs(double);
 
 extern double __enzyme_error_estimate(void *, ...);
 
+void enzymeLogError(double res, double err, const char *opcodeName,
+                    const char *calleeName, const char *moduleName,
+                    const char *functionName, const char *blockName) {
```

Comment:
For the test, can you have it increment a global counter, and test the counter below is set?

Reply:
Hey Billy, like [`b471d6e`](https://github.com/EnzymeAD/Enzyme/pull/1859/commits/b471d6ebe2a16b3815d89f9c127eaa98730557ce)?